test/CMakeLists.txt

```diff
@@ -106,6 +110,7 @@ add_fmt_test(enforce-checks-test)
 target_compile_definitions(enforce-checks-test PRIVATE
                            -DFMT_ENFORCE_COMPILE_STRING)
 
+# FIXME: never reached? CK
```

Comment:
What is it about?

Reply:
bash-5.2$ ninja -C build test   
ninja: Entering directory `build'
[0/1] Running tests...
Test project /Users/clausklein/Workspace/cpp/fmt/build
No tests were found!!!
bash-5.2$